server/container/guice/cassandra-guice/src/test/java/org/apache/james/DockerCassandraRule.java

```diff
@@ -0,0 +1,92 @@
+/****************************************************************
```

Comment:
Arf I made a reference to "cassandra.nodes" which is really introduced in the next PR, I should put it in the next PR